api/engine.go

```diff
@@ -181,29 +106,40 @@ const (
 	ExecutionInvalidTerminalBlock ExecutePayloadStatus = "INVALID_TERMINAL_BLOCK"
 )
 
+// PayloadID is an identifier of the payload build process
+type PayloadID [8]byte
```

Comment:
Instead you can import `beacon.PayloadID`. See https://github.com/ethereum/go-ethereum/blob/f0328f241b7c3def217b0c2dce1a1b297f979a37/core/beacon/types.go#L88 (the nice payload ID type is upstream now :) )

Reply:
Done! Thanks